Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -82,6 +86,45 @@ An impact defines the relation between an incident and a component. A component
 
 To reflect this, each component and incident can have a list of impacts, stating the type of impact and a reference to the incident or component, it refers to.
 
+Furthermore, a `SeverityValue` SHOULD be added to the `Impact` when referencing a component, to gauge the impact's severity on that component.
```

Comment:
Is `SHOULD` the right wording here? 
Is it optional but recommended to add the value to the impact, or should this be required? @snkaupe

Reply:
I'd prefer it being required.